client/src/main/scala/drt/client/services/handlers/ShiftsHandler.scala

```diff
@@ -6,25 +6,25 @@ import scala.scalajs.concurrent.JSExecutionContext.Implicits.queue
 import diode.data.{Pot, Ready}
 import diode.{ActionResult, Effect, ModelRW}
 import diode.Implicits.runAfterImpl
-import drt.client.actions.Actions.{AddShift, GetShifts, RetryActionAfter, SetShifts}
+import drt.client.actions.Actions.{GetShifts, RetryActionAfter, SetShifts}
 import drt.client.logger.log
 import drt.client.services.{AjaxClient, PollDelay, ViewMode}
 import drt.shared.Api
 
 import scala.concurrent.Future
 import scala.concurrent.duration._
 import scala.language.postfixOps
+import scala.scalajs.concurrent.JSExecutionContext.Implicits.queue
 
 class ShiftsHandler[M](viewMode: () => ViewMode, modelRW: ModelRW[M, Pot[String]]) extends LoggingActionHandler(modelRW) {
```

Comment:
Not sure what the difference this class is with FixedPointsHandler?

Reply:
shifts are available staff, fixed points are staff used up for a particular job, eg Roving officer or egates referral desk